tests/end-to-end/test-language-detect.bats

```diff
@@ -0,0 +1,52 @@
+setup() {
+  load "${BATS_LIBS_ROOT}/bats-support/load"
+  load "${BATS_LIBS_ROOT}/bats-assert/load"
+}
+
```

Comment:
I think you should leave a comment as to why these tests are commented out, so there is no confusion as to why if someone comes in after you. Also shouldn't `setup()` be commented out?